extension-push/src/java/com/defold/push/Push.java

```diff
@@ -351,10 +360,11 @@ public void scheduleNotification(final Activity activity, int uid, long timestam
         intent.setAction("uid" + uid);
         intent.putExtra(packageName + DEFOLD_NOTIFICATION, getLocalNotification(appContext, extras, uid));
 
-
-        PendingIntent pendingIntent = PendingIntent.getBroadcast(appContext, 0, intent, PendingIntent.FLAG_ONE_SHOT);
+        final int flags = createPendingIntentFlags(PendingIntent.FLAG_ONE_SHOT);
+        PendingIntent pendingIntent = PendingIntent.getBroadcast(appContext, 0, intent, flags);
         try {
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+            // from S the use of exact alarms requires additional permissions
```

Comment:
I don't think we want to add another permission?

https://developer.android.com/training/scheduling/alarms#exact-permission-declare

Reply:
it seems like it's not necessary.
We can make it optional using flag in game.project and Mustache template in AndroidManifest for that.